common/test/acceptance/tests/lms/test_lms_problems.py

```diff
@@ -412,3 +479,72 @@ def test_logout_cancel_no_redirect(self):
 
         self.assertTrue(problem_page.is_browser_on_page())
         self.assertEqual(problem_page.problem_name, 'TEST PROBLEM')
+
+
+# @attr('a11y')
+class CAPAProblemQuestionA11yTest(ProblemsTest):
+    """
+    TestCase Class for CAPA problem questions
+    """
+    def get_problem(self):
+        """
+        Problem structure.
+        """
+        xml = dedent("""
+            <problem>
+                <question>
+                    <p>question text</p>
+                    <stringresponse answer="A">
+                        <stringequalhint answer="C"><a href="#">aa bb</a> cc</stringequalhint>
+                        <textline size="20"/>
+                    </stringresponse>
+                    <demandhint>
+                      <hint>question 1 hint 1</hint>
+                      <hint>question 1 hint 2</hint>
+                    </demandhint>
+                </question>
+                <question>
+                    <p>That is the question</p>
+                    <multiplechoiceresponse>
+                      <choicegroup type="MultipleChoice">
+                        <choice correct="false">Alpha <choicehint>A hint</choicehint></choice>
+                        <choice correct="true">Beta</choice>
+                      </choicegroup>
+                    </multiplechoiceresponse>
+                    <demandhint>
+                      <hint>question 2 hint 1</hint>
+                      <hint>question 2 hint 2</hint>
+                    </demandhint>
+                </question>
+            </problem>
+        """)
+        return XBlockFixtureDesc('problem', 'Problem Question TEST', data=xml)
+
+    def get_num_problems(self):
+        """
+        Create two problems.
+        """
+        return 2
+
+    def test_questions_have_unique_ids(self):
+        """
+        Scenario: Verifies that each question has a unique id.
+        Given I am enrolled in a course.
+        And I visit a unit page with two CAPA problems where each problem has 2 questions
+        Then I check if questions have unique IDs
+        """
+        self.courseware_page.visit()
+        problem_page = ProblemPage(self.browser)
+
+        # Set the scope to the problem question
+        problem_page.a11y_audit.config.set_scope(
+            include=['div.question']
+        )
+
+        # Check only for duplicate ids
+        problem_page.a11y_audit.config.set_rules({
```

Comment:
Are there are lot of other existing issues with problems so that we can't just call check_for_accessibility_errors without limiting the ruleset? We don't typically write just scoped a11y tests, as we'd like to prevent new issues from creeping in.